changelog/12414.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
nit: I wonder if this is an enhancement instead of a bug? I _think_ the implementation before functioned as designed.

Reply:
I agree with that